app/components/LargeHeader.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 import { images } from '../assets';
+import { browserHistory } from 'react-router'
 
 
 function LargeHeader() {
+  var submitSearch = function() {
+    browserHistory.push({
+        pathname: '/resources',
+        query: { query: document.getElementById("srch-term").value }
```

Comment:
I think you can restrict the selector to the component rather than the full document. Also, should we validate the text? We may not want an empty search bar or a whitespace-only entry to submit